lib/ansible/modules/remote_management/hpilo/hpilo_boot.py

```diff
@@ -75,6 +75,12 @@
     - As a safeguard, without force, hpilo_boot will refuse to reboot a server that is already running.
     default: no
     choices: [ "yes", "no" ]
+  ssl_version:
+    description:
+      - Change the ssl_version used
+    default: TLSv1
+    choices: ['SSLv3','SSLv23','TLSv1','TLSv1_1','TLSv1_2']
```

Comment:
Another nitpick, the various items do not have to be quoted in YAML/JSON.
They have to be quoted for "yes" and "no" because these would otherwise be considered booleans.
Yes, YAML is tricky in many ways :-P